htdocs/multicurrency/multicurrency_rate.php

```diff
@@ -64,6 +64,10 @@
 $sortfield 			= GETPOST('sortfield', 'aZ09comma');
 $sortorder 			= GETPOST('sortorder', 'aZ09comma');
 $page = (GETPOST("page", 'int') ? GETPOST("page", 'int') : 0);
+// Prevents PHP Warnings
+$type = ''; 			// PHP Warning:  Undefined variable $type on line 440
```

Comment:
Can you remove all comment // PHP Warning ...
This is already provided into the PR title and we can't add such comment at each init of a variable (init should be the standard behaviour so no comment required).